src/solver/optimisation/opt_optimisation_lineaire.cpp

```diff
@@ -66,35 +65,15 @@ double OPT_ObjectiveFunctionResult(const PROBLEME_HEBDO* Probleme,
         return Probleme->coutOptimalSolution2[NumeroDeLIntervalle];
 }
 
-bool OPT_OptimisationLineaire(const OptimizationOptions& options, PROBLEME_HEBDO* problemeHebdo, AdqPatchParams& adqPatchParams,
-                              Solver::IResultWriter& writer)
+static bool runWeeklyOptimization(const OptimizationOptions& options,
```

Comment:
We could move the new methods together in the anonymous namespace above

Reply:
done